<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingListClass</class>
 <widget class="QMainWindow" name="ShoppingListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShoppingList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="ShoppinglistWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilterEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string> Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QListWidget" name="FilteredList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ShoppingList.qrc"/>
 </resources>
 <connections/>
</ui>
